library/entity/entity_events/src/main/java/org/quiltmc/qsl/entity_events/api/ServerPlayerEntityCopyCallback.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021-2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.entity_events.api;
+
+import net.minecraft.server.network.ServerPlayerEntity;
+
+import org.quiltmc.qsl.base.api.event.Event;
+import org.quiltmc.qsl.base.api.event.EventAwareListener;
+
+/**
+ * A callback which is called on the logical server when a player is copied.
+ * <p>
+ * Players are copied on death and when returning from the end through a portal. The {@code wasDeath} parameter can
```

Comment:
Small thing, maybe `wasDead` instead of `wasDeath`

Reply:
Actually, `wasDead` might be a bit confusing because in reality, it's trying to see if the cause was death, not if the player was dead

Reply:
> Actually, `wasDead` might be a bit confusing because in reality, it's trying to see if the cause was death, not if the player was dead

agreed.